src/index.ts

```diff
@@ -30,7 +30,7 @@ if (help) {
 const resolvedConfig = {
   ...config,
   command: mainOptions.command,
-  input: config.input.split(',')
+  ...(config.input ? { input: config.input.split(',') } : {}),
```

Comment:
Won't `input: config.input?.split(',')` suffice?

Reply:
No, as I mentioned in the PR description, the code you propose gives `input: undefined`.

And later in `flatFileConfig` and elsewhere, we do spreads, which means it overwrites the default configuration:
```ts
  const defaults = defaultConfig(projectType);
  const fileConfig = getGlobalConfig(inlineConfig.config || projectBasePath);
  const userConfig = { ...flatFileConfig(fileConfig), ...inlineConfig };
  const mergedConfig = { ...defaults, ...userConfig } as Config;
```

Here, if `defaults` have `{ input: ['app'] }` and `userConfig` `{ input: undefined }` the result of spread is  `{ input: undefined }`, this cause lot of trouble.

Reply:
I'll try to think of a cleaner way for this as it's also repeating in the `resolve-config.ts` file.

Reply:
Something doesn't make sense in what you are describing.
Let's say `config.input` is `undefined` then `config.input.split(',')` would have thrown. in what cases did you encounter that scenario? 
Generally speaking, I would resolve this issue where I'm resolving my config by filtering out `nil` values.

Reply:
Before that there was an `if`:
```
if (config.input) {
  config.input = config.input.split(',');
}
```
You removed this control in your commit (`infra/migrate-to-esm`), which can create an error.
<img width="708" alt="image" src="https://github.com/jsverse/transloco-keys-manager/assets/22544289/4b264dfe-d2a4-40aa-b175-84865739c3e5">
It's exactly the same for the `resolve-config.ts` file, before it was `if` you removed them and it changed the behavior.
<img width="926" alt="image" src="https://github.com/jsverse/transloco-keys-manager/assets/22544289/add3f60b-8049-47b8-8d3e-6326b84a01ed">

And about this syntax, is just a non-mutated es6 equivalent of the `if` from before.

Anyway, that's not the purpose of this PR, if you have refactors to do, I suggest you do them in another PR afterwards.

Reply:
As long as the specs are passing I'm ok with it, I'll do the refactor later